spec/domain/formations/squads_spec.exs

```diff
@@ -0,0 +1,41 @@
+defmodule Squadster.Domain.Formations.SquadsSpec do
+  use ESpec.Phoenix, async: true
+  use ESpec.Phoenix.Extend, :model
+
+  alias Squadster.Formations.Squad
+
+  describe "changeset" do
+    context "when params are valid" do
+      it "is valid" do
+        %{valid?: is_valid} = %{squad_number: "123456", class_day: :monday} |> Squad.changeset
+        expect is_valid |> to(be true)
+      end
+    end
+
+    context "when squad_member is not set" do
```

Comment:
Maybe make sence to move it under same `context`, something like `when wrong params passed`?

Reply:
makes sense, done